.github/workflows/test-monitor-flaky.yml

```diff
@@ -1,11 +1,15 @@
-name: Flaky Test Monitor
+# This workflow run ALL tests, including all tests that are skipped because they are flaky, as well as all the normal (non-skipped) tests.
+# This workflow is run less frequently because running flaky tests is problematic and causes errors.
+
+name: Test Monitor - Flaky
 
 on:
   schedule:
-    - cron: '0 */2 * * *' # every 2 hours
+    - cron: '0 */12 * * *' # every 12 hours
```

Comment:
changed to run every 12 hours instead of every 2 hours because this workflow often has problems / failures due to running flaky tests